crawler/crawler.go

```diff
@@ -146,3 +159,57 @@ func validateRemoteFile(data []byte, url string) error {
 
 	return publiccode.Parse(data, &pc)
 }
+
+// saveES save the chosen <file_name> in elasticsearch
+func saveToES(source, name string, data []byte) {
+	const (
+		index = "publiccode" // Elasticsearch index.
```

Comment:
index name must contains a timestamp to be unique in time

Reply:
ES index should be the timestamp of when crawler started or when single repository is checked or the date (DD-MM-YYYY) of the single check??

Reply:
ok here `publiccode` is the name of the alias. In the index creation you have to chose a random name (a timestamp maybe) and set the alias `publiccode` to it. If the index already exists and this execution is a recovery (because you have some data in Redis) you have to continue using this index, if this is as new execution you have to create a new index and at the end of the process, switch the alias to point to this index